eessi/testsuite/tests/apps/espresso/benchmarks.csv

```diff
@@ -0,0 +1,34 @@
+"mode","cores","mpi.x","mpi.y","mpi.z","particles","mean","std"
```

Comment:
This whole file seems unused? I've noticed that the default for the `--output` argument of `madelung.py` is `benchmarks.csv`, but that argument is unused. Also, since this is not in the `src` dir, it's not even copied to the ReFrame staging dir, so I'm pretty sue this file is not needed/being used right now?